modules/ROOT/pages/to-download-and-install-studio-lx.adoc

```diff
@@ -3,27 +3,39 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To support some of Studio Themes in Linux, it is recommended to have GTK version 2 installed.
+== Prerequisites
 
-* Before downloading and launching Anypoint Studio, make sure you have the Java SE JDK 8 installed.
+* Ensure that you have the Java SE JDK 8 installed and configured.
+* To support some of Studio Themes in Linux, it is recommended to have GTK version 2 installed.
+* Download the Anypoint Studio installation file from the https://www.mulesoft.com/lp/dl/studio[Anypoint Studio Download Site].
+
+[NOTE]
```

Comment:
So, this could remain a Note, or you could make it a plain text introduction between the Prerequisites title and the bullet list, or you could make it the first bullet:

* If you are using Anypoint Studio 7.x, your project must use Mule runtime engine 4.x.

I think I like this suggestion best: it is a prerequisite when stated this way. I don't think that we have to explain why.